control/control.rnc

```diff
@@ -186,10 +186,14 @@ default_ntp_setup =			element default_ntp_setup { BOOLEAN }
 enable_kdump =				element enable_kdump { BOOLEAN }
 require_registration =			element require_registration { BOOLEAN }
 show_drivers_info =			element show_drivers_info { BOOLEAN }
-# Self-update (FATE#319716).
+# Self-update URL (FATE#319716).
+## How it works?
+## here are two related keys for self update - URL and ID.
+## The ID is used to ask SCC for update URL.
```

Comment:
More picky questions: do we know the possible values/format of the ID? Or the authoritative place where it is specified?

Reply:
@mvidner it is defined by SCC which should know which one accept. I plan to find value experimentally ( with SLES as starting point ).